experiments/run_experiment.py

```diff
@@ -195,7 +199,7 @@ def initialize_recourse_method(
 
 results = pd.DataFrame()
 
-path = args.path
+path =  "results2.csv"
```

Comment:
Lets stick with the argument path here instead.

Reply:
Thanks! removed